src/Evolution/Systems/GeneralizedHarmonic/BoundaryConditions/BjorhusImpl.hpp

```diff
@@ -0,0 +1,48 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/Tensor/TypeAliases.hpp"
+
+/// \cond
+class DataVector;
+namespace gsl {
+template <class T>
+class not_null;
+}  // namespace gsl
+/// \endcond
+
+namespace GeneralizedHarmonic::BoundaryConditions {
+namespace detail {
+// @{
+/*!
```

Comment:
things in `detail` namespaces don't render the dox -- maybe it's best to move it out of the detail namespace, or perhaps rename the namespace to something like `Bjorhus` or `BjorhusImpl`?

Reply:
okay, renamed `detail` to `Bjorhus`